### PR TITLE
[SPARK-40124][SQL][TEST] Update TPCDS v1.4 q32 for Plan Stability tests

### PR DESCRIPTION
**Pull request by mskapilks**
### What changes were proposed in this pull request?
Fix query 32 for TPCDS v1.4

### Why are the changes needed?
Current q32.sql seems to be wrong. It is just selection `1`.
Reference for query template: https://github.com/databricks/tpcds-kit/blob/eff5de2c30337b71cc0dc1976147742d2c65d378/query_templates/query32.tpl#L41

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Test change only.

### CODE REVIEW

**Review by wangyum — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

cc @viirya , @huaxingao , @sunchao , @parthchandra , @kazuyukitanimura

**Review by Swinky — APPROVED** [review]

**Review by viirya — APPROVED** [review]

**Review by huaxingao — APPROVED** [review]

File: sql/core/src/test/resources/tpcds/q32.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SELECT 1 AS `excess discount amount `
+SELECT sum(cs_ext_discount_amt) AS `excess discount amount `
```
Comment (huaxingao) [inline sql/core/src/test/resources/tpcds/q32.sql:1]: The change looks good, but do we need the empty space after `amount`?

**Comment by kazuyukitanimura** [comment]
I think we should update the benchmark results too `sql/core/benchmarks/TPCDSQueryBenchmark-***-results.txt`

**Review by HyukjinKwon — APPROVED** [review]

File: sql/core/src/test/resources/tpcds/q32.sql (hunk 1 above)
Comment (mskapilks) [reply sql/core/src/test/resources/tpcds/q32.sql]: Removed the extra space

**Comment by mskapilks** [comment]
> I think we should update the benchmark results too `sql/core/benchmarks/TPCDSQueryBenchmark-***-results.txt`

Have updated the benchmark files @kazuyukitanimura

**Review by kazuyukitanimura — APPROVED** [review]
LGTM (pending CI) Thank you for updating the benchmark files

**Comment by dongjoon-hyun** [comment]
Merged to master. Thank you all.

@mskapilks , unfortunately, this should be done separately for all live branches. Could you make backporting PRs for branch-3.3/branch-3.2/branch-3.1?

**Comment by dongjoon-hyun** [comment]
I added your `kapilks_ms` ID to Apache Spark contributor group and assigned SPARK-40124 to you.
Welcome to the Apache Spark community, @mskapilks .

**Pull request closed without merging**